packages/base/src/trace-manager.ts

```diff
@@ -1,15 +1,16 @@
 import { Trace } from 'tsp-typescript-client/lib/models/trace';
-import { TspClient } from 'tsp-typescript-client/lib/protocol/tsp-client';
 import { Query } from 'tsp-typescript-client/lib/models/query/query';
 import { OutputDescriptor } from 'tsp-typescript-client/lib/models/output-descriptor';
 import { TspClientResponse } from 'tsp-typescript-client/lib/protocol/tsp-client-response';
 import { signalManager } from './signals/signal-manager';
+import { TspFrontendClient } from './tsp-frontend-client';
+import { TspClient } from 'tsp-typescript-client';
```

Comment:
Minor; I would leave this import line unchanged at its original position.
```suggestion
import { TspClient } from 'tsp-typescript-client/lib/protocol/tsp-client';
```